src/engine/world/actor/dialogue.ts

```diff
@@ -364,16 +364,17 @@ function parseDialogueTree(player: Player, npcParticipants: NpcParticipant[], di
                     }
                 }
 
+                console.log('npc:', npc);
+
```

Comment:
Not to be that guy, bur is the console.log needed?

Reply:
Nope, I'm just dumb and forgot to remove it, good catch :)